keyboards/bbrfkr/dynamis/readme.md

```diff
@@ -0,0 +1,19 @@
+# dynamis
+
+![dynamis](https://i.imgur.com/q2C3D2u.jpg)
+
+Gasket mounted 65% row staggered keyboard integrated a trackball
+
+* Keyboard Maintainer: [bbrfkr](https://github.com/bbrfkr/)
+* Hardware Supported: dynamis PCBs
+* Hardware Availability: http://github.com/bbrfkr/keyboards
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
I modified readme!